resources/template.docker-compose.yml

```diff
@@ -0,0 +1,19 @@
+# Please refer https://docs.docker.com/compose to learn more about Docker Compose.
+
+# This is a sample docker-compose file with two services
+# 1. yourwebapp is an sample web service where the docker container image will be built from the Dockerfile as
+#    part of starting the compose.
+# 2. redis is an existing image hosted in docker hub.
+version: '3.4'
+
+services:
```

Comment:
Note that the extension gets packaged pretty differently for prod (webpack) vs. when you F5. I think it's OK, but double check that this file ends up in the VSIX like we expect.

Reply:
We already use GetTargetPath.proj from resources folder. This is no different than the other file right?

Reply:
For the vsdbg script, at one point I had to put it in a manifest or move it or something like that; I can't recall.